source/components/GroupsPage.js

```diff
@@ -95,7 +97,11 @@ class GroupsPage extends Component {
                                             )
                                         }
                                     >
-                                        <Text>{group.title}</Text>
+                                        <Text>
+                                            {rawGroupIsTrash(group)
+                                                ? this.props.t("group.trash")
```

Comment:
I don't think we should do this. Titles should be pure and used entirely as the data within the vault - 1:1. Translating that is not a great idea. I would suggest that we translate this for when we **create**  these groups at vault creation.

Reply:
Ok. I removed both "Trash-Group" translations. It should best happen in this method, right? https://github.com/buttercup/buttercup-core/blob/7c60c6ed2c656790bbf23f42bfe20fc3942932a2/source/node/Archive.js#L295